webapp/components/_new/generic/PaginationButtons/PaginationButtons.vue

```diff
@@ -1,31 +1,56 @@
 <template>
   <div class="pagination-buttons">
     <base-button
-      @click="$emit('back')"
       :disabled="!hasPrevious"
       icon="arrow-left"
       circle
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-05-13T17:53:24Z" title="Wednesday, May 13th 2020, 7:53:24 pm +02:00">May 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,32 +1,56 @@
 <template>
   <div class="pagination-buttons">
     <base-button
+      v-if="hasPreviousResult"
       @click="$emit('back')"
-      :disabled="!hasPrevious"
+      :disabled="!hasPreviousResult"
```


As we use `v-if` disabled is not necessary